timescaledb/quick-start/ruby.md

```diff
@@ -4,727 +4,708 @@ excerpt: Get started with TimescaleDB for a Ruby application
 keywords: [Ruby]
 ---
 
-# Quick Start: Ruby and TimescaleDB
+import Intro from "versionContent/_partials/_quickstart-intro.mdx";
 
-## Goal
+# Ruby quick start
 
-This quick start guide is designed to get the Rails developer up
-and running with TimescaleDB as their database. In this tutorial,
-you'll learn how to:
+<Intro />
 
-*   [Connect to TimescaleDB](#connect-ruby-to-timescaledb)
-*   [Create a relational table](#create-a-relational-table)
-*   [Generate a Hypertable](#generate-hypertable)
-*   [Insert a batch of rows into your Timescale database](#insert-rows-into-timescaledb)
-*   [Execute a query on your Timescale database](#execute-a-query)
+This quick start guide walks you through:
+
+*   [Connecting to TimescaleDB][connect]
+*   [Creating a relational table][create-table]
+*   [Creating a hypertable][create-hypertable]
+*   [Inserting data][insert]
+*   [Executing a query][query]
 
 ## Prerequisites
 
-To complete this tutorial, you need a cursory knowledge of the Structured Query
-Language (SQL). The tutorial walks you through each SQL command, but it is
-helpful if you've seen SQL before.
+Before you start, make sure you have:
+
+*   Installed TimescaleDB. For more information, see the
+    [installation documentation][install]. Make a note of the `Service URL`,
+    `Password`, and `Port` in the TimescaleDB Cloud instance that you created.
+*   Installed [Rails][rails-guide].
+*   Installed [psql to connect][psql-install] to TimescaleDB.
+
+<highlight type="cloud" header="Run all tutorials free" button="Try for free">
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all the tutorials and run
+a few test projects of your own.
+</highlight>
```

Comment:
```suggestion
<Highlight type="cloud" header="Run all tutorials free" button="Try for free">
Your Timescale Cloud trial is completely free for you to use for the first
thirty days. This gives you enough time to complete all the tutorials and run
a few test projects of your own.
</Highlight>
```

(almost) all component names are highlighted now, here's the Slack thread about it: https://iobeam.slack.com/archives/C03H4440ZQC/p1677024093700659

I'm guessing I probably made the change while you were in the middle of working on this :)